doc/api/http2.md

```diff
@@ -24,8 +24,9 @@ be emitted either by client-side code or server-side code.
 
 ### Server-side example
 
-The following illustrates a simple, plain-text HTTP/2 server using the
-Core API:
+The following illustrates a simple HTTP/2 server using the Core API.
+Since no browsers support [unencrypted HTTP/2][HTTP/2 Unencrypted],
+the use of [`http2.createSecureServer()`][] is preferred.
 
```

Comment:
I would say something like...

```
Since there are no browsers known that support [unencrypted HTTP/2](HTTP/2 Unencrypted),
the use of [`http2.createSecureServer()`][] is necessary when communicating with browser
clients.
```

Using Unencrypted HTTP/2 would actually be the preference when running purely within internal environments with non-browser clients.

Reply:
Okie dokie 👍